src/zcl/definition/cluster.ts

```diff
@@ -4085,7 +4085,18 @@ const Cluster: {
                 ],
             },
         },
-    },    
+    },
+    develcoSpecificAirQuality: {
+        ID: 0xFC03,
+        attributes: {
```

Comment:
Since this is a manufacturer specific cluster please add the manufacturer code, example: https://github.com/Koenkk/zigbee-herdsman/blob/a6180afd31c6e125be615bc77cf1142977c868b2/src/zcl/definition/cluster.ts#L3749

Reply:
Added